### ISSUE TITLE
Concurrency

### ISSUE THREAD

**Issue by andyskw** [nodebp/nodebp#37]
Concurrency came up for a couple of seconds in the last Nodebp, and as I haven't found anything in the closed issues list (and also in open issues list), I decided to post an issue. :)

I'd suggest a topic on concurrency in NodeJS, with something like this:
- What is concurrency? Why is it important to you?
- Process vs thread based concurrency
- The importance of CPU utilization
- Concurrency possibilities in node
- Cluster API example
  - basic app setup with cluster and/or strong-cluster-control
  - IPC
  - error handling
  -  shiny, real-time benchmarks are welcomed, with AHA! moments, etc. :)
-  etc. ?

What do you think? I am not sure if it really is a topic to be covered here - also it's not clear, if it's an expert or a beginner topic [its deep enough to be any of them, depending on the requirements :D], but as it came up last time, it seems so, that it may be somewhat interesting. 

Also, in the long term, I am interested in giving a speech on this topic in the mid run (in April?), but it's also ok if someone wants to take the topic. :)

What do you think?

**Comment by gergelyke** [comment]
Hey,

thanks for the proposal! Sure, April sounds good! In April @EggDice will be in charge to organise Nodebp, so he will be your guy ;)

**Comment by nightw** [comment]
I like the idea. :)

**Comment by gergelyke** [comment]
@nightw what a surprise :D If i recall correctly, you guys work together :P

**Comment by nightw** [comment]
Yeah. :D And I like the topic, since I think it's important performance-wise and I'm mostly interested in running Node.js and the aspects around that like performance. (And @andyskw would not do this presentation at Statzup, since there is only about 3-4 people interested in it, so the meetup is a great excuse for doing a much better/deeper presentation).

**Comment by gergelyke** [comment]
Agreed! I hope to hear it in April! :)

On Tue, Jan 27, 2015 at 11:55 AM, Gergely Pal David <
notifications@github.com> wrote:

> Yeah. :D And I like the topic, since I think it's important
> performance-wise and I'm mostly interested in running Node.js and the
> aspects around that like performance. (And @andyskw
> https://github.com/andyskw would not do this presentation at Statzup,
> since there is only about 3-4 people interested in it, so the meetup is a
> great excuse for doing a much better/deeper presentation).
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nodebp/nodebp/issues/37#issuecomment-71626919.

## 

_Gergely Nemeth_
Phone: + 36 30 926 1231
Twitter: @nthgergo

**Comment by EggDice** [comment]
Just hit me on March and we will discuss!
